### ISSUE TITLE
Screen junk with latest update

### ISSUE THREAD

**Issue by StanleyKnife** [uetchy/Polyglot#88]
**Describe the bug**
When Polyglot is loaded, a pages with Arabic text shows an always-on grey box. Text highlighted for translation shows a spinning box top left.

**Screenshots**

Always-on box at top, highlighted translation bottom.

**Version (please complete the following information):**

- Version 3.2

![polyglot issue](https://user-images.githubusercontent.com/3235450/105195810-e4cc0380-5b32-11eb-81c7-73d28e2fa826.jpg)

**Comment by gingerbeardman** [comment]
I'm not able to reproduce

using https://ar.wikipedia.org/wiki/الصفحة_الرئيسية as a test

**Comment by StanleyKnife** [comment]
Sorry - the site is [fm.gov.om](https://fm.gov.om/ministry-ar/about-us-ar/key-figures/?lang=ar) and I'm finding this issue each time I turn Polyglot back on. My co-workers are finding the same issue.

**Comment by StanleyKnife** [comment]
OK this is really odd. I've found that when I translated some text while editing the page (and thanks for that new ability, by the way) the following code was inserted in the panel I was editing:

<div id="polyglot__panel" style="left: 10px; top: 86.4375px;">
<h3 class="polyglot__inner">تهتدي الدبلوماسية العمانية بالفكر السامي لحضرة صاحب الجلالة السلطان المعظم ورؤيته وتوجيهاته، وتلتزم بالمبادئ التي ينص عليها النظام الأساسي للدولة.</h3>
</div>

**Comment by gingerbeardman** [comment]
that's the problem!

What type of edit control is the page using?

**Comment by StanleyKnife** [comment]
It's the Avada theme, which has its own editing environment. 

Previously Polyglot was unable to translate Arabic text within the editor window, so it wasn't an issue. I think I should just remember not to use it here.

By the way, Polyglot is a fantastic tool. Exactly what I need. Thank you!

**Comment by gingerbeardman** [comment]
Oh, and I use CleanMyMac to trim language files which still works well.

<img width="846" alt="Screen shot 2021-01-21 at 11 21 27" src="https://user-images.githubusercontent.com/49612/105344449-df29f880-5bda-11eb-8dab-7ac50269e5dd.png">

**Comment by gingerbeardman** [comment]
I have also had Polyglot leaving text inside of text fields after selecting text in text field. I was editing an item on archive.org at the time.

**Comment by StanleyKnife** [comment]
Yes, it seems to happen when you translate text inside an edit field. Works fine on non-editing web pages.

**Comment by gingerbeardman** [comment]
I thought that in https://github.com/uetchy/Polyglot/issues/28 translation was disabled in text field, because it's most likely the words in there are either in your own language, or words I am typing myself!

**Comment by StanleyKnife** [comment]
No, you can still translate - but the extra code is inserted when you do.

**Comment by gingerbeardman** [comment]
I am getting this in every text area:
- Wordpress
- Wikipedia
- Internet Archive

**Comment by uetchy** [comment]
Actually I couldn't reproduce this bug. Could you give me a complete and precise procedure to reproduce the bug?

**Comment by gingerbeardman** [comment]
Sure!

First you nee to create a page on internet archive (see below), or a post on a wordpress blog, or edit a wikipedia page.

Then, simply select text in the text field:
1. popup html is added as text inside the text field

This is a problem: if the popup is not dismissed the article can be saved and the HTML for the Polyglot popup will be saved with the contents of the text field!

Also:

1. goto https://archive.org/create/
2. click: Upload Files
3. select a local file to upload
4. click: description field
5. type some description
6. select the text you have just typed

Problem:
1. multiple popups

**Comment by gingerbeardman** [comment]
Similarly, with WordPress:

<img width="588" alt="Screen shot 2021-03-18 at 10 43 27" src="https://user-images.githubusercontent.com/49612/111613872-e3d3ed00-87d6-11eb-9bc9-daf704ad03a7.png">

I do not have time to show Wikipedia, but the process is the same.

Basically, do something on the page to change DOM before popup is dismissed and we lose control of the popup and its HTML code.

**Comment by uetchy** [comment]
<img width="869" alt="Screen Shot 2021-03-19 at 5 42 17 PM" src="https://user-images.githubusercontent.com/431808/111753637-74472600-88da-11eb-8e59-9cc3fc7e3fac.png">

I can confirm.
That's a bit tricky problem.
Normally Polyglot can suppress instant translation when the selected text is in a textarea or input tag, but in this case the editor is just a `body` inside `iframe` that Polyglot cannot tell it's an editable text.

**Comment by uetchy** [comment]
Disabling iframe handling is the easiest route, but I don't want to do it because it also prevents me from translating live chat comments on YouTube. We need to find another solution to this.

**Comment by gingerbeardman** [comment]
I'll also think about this problem.

**Comment by gingerbeardman** [comment]
Just to make some notes on possible exclusion list of selectors:

- Internet Archive: `div.wysiwyg iframe`
- WordPress: `div.mce-tinymce`
- Wikipedia: `div.ve-ce-surface`
